ogusa/SS.py

```diff
@@ -235,23 +235,65 @@ def SS_solver(bmat, nmat, r, BQ, T_H, factor, Y, p, client,
               fsolve_flag=False):
     '''
     Solves for the steady state distribution of capital, labor, as well
-    as w, r, T_H and the scaling factor, using functional iteration.
```

Comment:
@duncanhobbs Here you are deleting the wrong thing when there was a merge conflict.  You want to keep the HEAD in this conflict case (here, that means the docstrings that follow the Google docstring style).